lua/elixir/projectionist/init.lua

```diff
@@ -102,6 +102,20 @@ local config = {
         "end",
       },
     },
+    ["lib/**/live/*_live.html.leex"] = {
+      type = "leex",
+      alternate = "test/{dirname}/live/{basename}_live_test.exs",
+      template = {
+        "",
+      },
+    },
+    ["lib/**/live/*_live.html.heex"] = {
```

Comment:
I don't think people name their templates suffixed with `_live`.

I have no memory of ever doing that or of having seen the generators do that. Where have you seen this convention?

<img width="838" alt="image" src="https://user-images.githubusercontent.com/5523984/236559025-5a1352e8-e072-4495-a468-bd5b39784ada.png">

Reply:
It's unclear to me what the convention is at this time.

Reply:
Hm in this example the liveview files themselves tend to be suffixed with `_live.ex` as seen In the screenshot you posted with `/live/home_live.ex`

also seen at

https://github.com/fly-apps/live_beats/tree/master/lib/live_beats_web/live

The most compelling example is seen in the official Phoneix LiveView documentation for [Colocating Templates](https://hexdocs.pm/phoenix_live_view/Phoenix.LiveView.html#module-colocating-templates)

> For larger templates, you can place them in a file in the same directory and same name as the LiveView. For example, if the file above is placed at lib/my_app_web/live/thermostat_live.ex, you can also remove the [render/1](https://hexdocs.pm/phoenix_live_view/Phoenix.LiveView.html#c:render/1) definition above and instead put the template code at lib/my_app_web/live/thermostat_live.html.heex.

Reply:
> Hm in this example the liveview files themselves tend to be suffixed with _live.ex as seen In the screenshot you posted with /live/home_live.ex
> 
> also seen at
> 
> https://github.com/fly-apps/live_beats/tree/master/lib/live_beats_web/live

I'm referring to the template naming.

> The most compelling example is seen in the official Phoneix LiveView documentation for [Colocating Templates](https://hexdocs.pm/phoenix_live_view/Phoenix.LiveView.html#module-colocating-templates)
>
> > For larger templates, you can place them in a file in the same directory and same name as the LiveView. For example, if the file above is placed at lib/my_app_web/live/thermostat_live.ex, you can also remove the render/1 definition above and instead put the template code at lib/my_app_web/live/thermostat_live.html.heex.

I think this might be an outdated example, as I think the current way to do the templates is just with components, using the `embed_templates` macro

Reply:
If you'd like I could remove the `_live` portion and it would effectively allow that to be optional since `_live` is part of the filename if present.

Reply:
makes me wonder if we should make the live view and live view test projections also drop that static convention.

Reply:
I could do that!

It would change the `Elive blog_web/post` command to `Elive blog_web/post_live` but make it more applicable to people who use different conventions so that seems like a worthy tradeoff.

Reply:
I think that makes sense. Thanks. 

I probably won't get to testing this until next week. 

Could you also (can be a separate PR) add some examples to the README?